ts/masters/modules/map.src.ts

```diff
@@ -21,4 +22,8 @@ import '../../Series/MapPoint/MapPointSeries.js';
 import '../../Series/MapBubble/MapBubbleSeries.js';
 import '../../Series/Heatmap/HeatmapSeries.js';
 import '../../Extensions/GeoJSON.js';
-import '../../Maps/Map.js';
+import M from '../../Core/Chart/MapChart.js';
```

Comment:
I would prefer if the imports in masters are more descriptive, because later on all imports are named, and a single letter will not work then anymore. I suggest in this case here `MapChart`.